buildme

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#source /etc/profile.d/z00_lmod.sh
+#module load cuda/10.2
+#module load gcc/8.3.1
+module load cmake/3.23.1 # to avoid cmake FindMPI problem
+
+set -x
+
+installdir=`pwd`/install
+#rm -rf $installdir
+
+# download SCR-v3.0.1 release
+if [ ! -f scr-v3.0.1.tgz ] ; then
+  wget https://github.com/LLNL/scr/releases/download/v3.0.1/scr-v3.0.1.tgz
+  if [ ! -d scr-v3.0.1 ] ; then
+    tar -xzf scr-v3.0.1.tgz
+  fi
+fi
+
+# build SCR-v3.0.1 in debug mode
+pushd scr-v3.0.1
+  installdir_scr=`pwd`/install
+  rm -rf $installdir_scr
+
+  rm -rf build
+  mkdir build
+  pushd build
+    cmake \
+      -DCMAKE_INSTALL_PREFIX=$installdir_scr \
+      -DCMAKE_BUILD_TYPE=Debug \
+      -DSCR_RESOURCE_MANAGER=LSF \
+      -DENABLE_PDSH=OFF \
+      ..
+    make -j
+    make install
+  popd
+popd
+
+# build ADIOS2 and point to the above SCR installation
+rm -rf build
+mkdir build
+cd build
+cmake \
```

Comment:
I had to load gcc to be able to build ADIOS2 (I did not load gcc to be able to build SCR)

Reply:
Yes, I think I had to do the same.